docs/topics/gradle/get-started-with-jvm-gradle-project.md

```diff
@@ -1,48 +1,53 @@
 [//]: # (title: Get started with Gradle and Kotlin/JVM)
 
-This tutorial demonstrates how to use IntelliJ IDEA and Gradle for creating a console application.
+This tutorial demonstrates how to use IntelliJ IDEA and Gradle to create a JVM console application.
 
 To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).
 
 ## Create a project
 
 1. In IntelliJ IDEA, select **File** | **New** | **Project**.
-2. In the panel on the left, select **New Project**.
+2. In the panel on the left, select **Kotlin**.
```

Comment:
This UI is only available since 241 AFAIU. Should we explicitly mention it or we assume this the default option now?